decidim-core/app/views/decidim/participatory_process_steps/_timeline.html.erb

```diff
@@ -0,0 +1,7 @@
+<ol class="timeline">
+  <% past_step = true %>
+  <% current_participatory_process.steps.each do |step| %>
+    <%= render step, step_status_class: step_class(step, past_step)  %>
+    <% past_step = false if step.active? %>
```

Comment:
Shit, I didn't read this https://github.com/AjuntamentdeBarcelona/decidim/pull/518#discussion-diff-96421697R2